internal/utils/numbers.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import (
+	"math/big"
+
+	"github.com/volatiletech/sqlboiler/v4/types"
+)
+
+func SafeNullDecimalToInt(num types.NullDecimal) *big.Int {
```

Comment:
I don't know about safe. It's a converter with a default—could we say that? `NullDecimalToIntDefaultZero`?